elements/pfe-jump-links/demo/index.html

```diff
@@ -57,36 +56,40 @@ <h1><code>pfe-jump-links</code></h1>
       position:sticky;
       top: 0;
     }
+    body {
+      font-family: 'Red Hat Text', sans-serif;
```

Comment:
@Djfaucette Are you loading the Red Hat font on the demo page?

Reply:
Yep.